src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -686,24 +693,72 @@ public void onFileChange(File file) {
         }
     }
 
-    private void buildDockerImage() throws PluginExecutionException {
+    private List<String> readDockerfile(File dockerfile) throws PluginExecutionException {
+        // Convert Dockerfile to List of strings for each line
+        List<String> dockerfileLines = null;
+        try {
+            dockerfileLines = Files.readAllLines(dockerfile.toPath());
+        } catch (IOException e) {
+            error("Failed to read Dockerfile located at " + dockerfile);
+            //TODO: what direction should we give the user here?
+            throw new PluginExecutionException("Could not read Dockerfile: " + dockerfile, e);
+        }
+        return dockerfileLines;
+    }
+
+    private List<String> removeWarFileLines(List<String> dockerfileLines) {
+        List<String> warFileLines = new ArrayList<String>();
+        for (String line : dockerfileLines) {
+            // Remove white space from the beginning and end of the line
+            String trimLine = line.trim();
+            if (!trimLine.startsWith("#") && trimLine.contains(".war")) {
+                // Break the Dockerfile line down into segments based on any amount of whitespace.
+                // The command must be to the left of any comments.
+                String[] cmdSegments = trimLine.split("#")[0].split("\\s+");
+                // if the line starts with COPY and the second to last segment ends with ".war", it is a WAR file COPY line
+                if (cmdSegments[0].equals("COPY") && cmdSegments[cmdSegments.length - 2].endsWith(".war")) {
```

Comment:
Docker allows the command to be lower case: "The instruction is not case-sensitive. However, convention is for them to be UPPERCASE to distinguish them from arguments more easily." Also, Windows allows the .WAR file name to be upper case so we should use equalsIgnoreCase().